### PR TITLE
Update Algolia autocomplete styles

### PR DESCRIPTION
**Pull request by Aym-n**
_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

**Comment by Aym-n** [comment]
@Scotchester Can you please review this PR?

File: docs/_static/css/docsearch.overrides.css [hunk 1]
```diff
@@ -28,3 +28,9 @@ a.algolia-docsearch-suggestion {
 body.theme-dark #search-form .algolia-autocomplete > .form-control {
   background: #1b1238 none repeat scroll 0% 0% padding-box !important;
 }
+
+/* overide algolia search default styles to prevent overflow on mobile devices */
+span#algolia-autocomplete-listbox-0 {
```
Comment (lb-) [inline docs/_static/css/docsearch.overrides.css:33]:
```suggestion
#algolia-autocomplete-listbox-0 {
```

No need to scope to the span here

**Review by lb- — CHANGES_REQUESTED** [review]
@Aym-n thank you for taking the time to investigate this.

These styles fix for small devices (stop the content from overflowing to the left), but actually breaks the intended behaviour of the content overflowing the TOC container on large devices.

## Screenshots

Left is this PR, Right is current state - see on mobile it looks better but on larger devices it is now too thin.

**Small**
Screen Shot 2023-01-21 at 7 40 59 am


**Large**
Screen Shot 2023-01-21 at 7 41 51 am

**Comment by Aym-n** [comment]
@lb- Should i use css media queries to apply the `#algolia-autocomplete-listbox-0` style rules only for mobile devices?

**Comment by lb-** [comment]
Good idea. Thanks @Aym-n

**Comment by Aym-n** [comment]
@lb-  do i need to change anything else? is this issue resolved?

**Comment by lb-** [comment]
Still on my radar. Looks like you have accidentally included some other commits. I've triggered the runner though.

**Review by lb- — APPROVED** [review]
I have isolated the commit that fixes the issue and validated it works as expected, thank you @Aym-n  - I will merge this in.

A note for next time, have a read of our WIP new contributors guide for some additional tips for getting a good PR ready.

One key item is to create a branch on your repo for your changes, it will help you manage the fixes in isolation of the `main` branch.

https://github.com/wagtail/wagtail/issues/10070

**Pull request merged**